src/operations/blas3/gemm_load_store.hpp

```diff
@@ -69,51 +71,49 @@ struct Packetize {
    * @tparam internal True if the current block is internal and no bounds
    * checking is required.
    * @tparam ld The leading dimension of the destination memory. */
-  template <bool trans, bool internal, int ld, typename packet_t = PacketType,
-            typename SrcPointerType, typename DestPointerType, typename index_t,
-            typename EdgePredicate>
+  template <bool trans, bool internal, index_t ld,
+            typename packet_t = PacketType, typename SrcPointerType,
```

Comment:
Was `packet_t` ever not `PacketType`? I thought you'd removed this parameter (but maybe I'm remembering a change from somewhere else).

Reply:
Ah yes, that change is in the subsequent PR I'll be putting up!